### DISCUSSION TITLE
Save Connection As

### DISCUSSION THREAD

**Discussion by ismaelbalkhi** [Eugeny/tabby#5686] [General]
Hello,

I love Tabby! Now that that is out of the way. I constantly SSH into hundreds of hosts, it would be nice if I could **'Right-Click'** on an established connection and **'Save-As'** a profile along with the login credentials I used. Right now, this easy method is not available. I have to go, one-by-one into Settings to create individual profiles which isn't feasible in the long run for so many connections. Also, does anyone know of a nice Sessions Manager within Tabby that could handle hundreds of saved profiles and be able to organize/sort them? 

Currently, I quickly connect via myName@10.10.10.10:22, but I don't see a way to save it efficiently.

Thanks!

**Comment by Eugeny** [comment]
There isn't a way to do this right now, but I've set up #5688 - should be pretty easy to do!

As for the profile manager, I think it'd be more productive to just list what you'd like to see improved in the profile settings tab :)

**Reply by ismaelbalkhi** [reply]
Thanks @Eugeny ! Much appreciated! This is a great Terminal Emulator that doesn't look like Windows 3.0.
